app/assets/stylesheets/components/_theme.css.scss

```diff
@@ -116,6 +116,10 @@ body {
     @include delta;
     line-height: 0.55;
   }
+
+  a {
```

Comment:
I think there is no difference, it's more a question of what makes more sense, for me I think here is a good place to set the color of the links, I would left the reset for things like margins, borders and so on